Integrations/integration-Fortigate.yml

```diff
@@ -0,0 +1,1381 @@
+description: Manage FortiGate Firewall
+configuration:
+- display: Server URL (e.g. 192.168.0.1)
```

Comment:
If `https://...` is required please add it

Reply:
I honestly can't tell if its required, not by any indication that I got from docs, will ask Arian.